### DISCUSSION TITLE
Rearrange columns in a simpler way

### DISCUSSION THREAD

**Discussion by shrivatsahosabettu** [dbeaver/dbeaver#19060] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Its really hard to re-arrange the columns from the result set by drag and drop one by one or I have write the SQL according to the column arrange I required.

**Describe the solution you'd like**
There should be a simpler way similar to filter, arrange columns and from the Metadata of the table we can select the columns by mentioning the columns which I need as 1, 2, 3, 4.. so on and the rest of the columns can follow the order of 1, 2, 3, 4..
Probably in the side panel section we can add the feature to re-arrange the columns.

**Describe alternatives you've considered**
Drag and drop each column to re-arrange or write the SQL in the require columns I need.

**Comment by E1izabeth** [comment, marked as answer]
You can do it via filter settings panel.
![image](https://user-images.githubusercontent.com/28875055/218673794-55a28250-16fa-4596-9eda-49e4ae061998.png)
![image](https://user-images.githubusercontent.com/28875055/218673925-097286c6-2a40-45b7-bd33-be508d7b5864.png)

Does it satisfy your needs?

**Comment by shrivatsahosabettu** [comment]
That's awesome.. thank you so much.
